src/main/resources/com/hpe/application/automation/tools/run/SonarOctaneListener/config.jelly

```diff
@@ -0,0 +1,27 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:s="/lib/samples">
+
+  <!--
```

Comment:
Is there a need for this file?
If so remove commented out code and add the license.

Reply:
fixed